src/power_grid_model/validation/validation.py

```diff
@@ -806,3 +809,20 @@ def validate_fault(data: SingleDataset) -> List[ValidationError]:
     errors += all_enabled_identical(data, "fault", "fault_type", "status")
     errors += all_enabled_identical(data, "fault", "fault_phase", "status")
     return errors
+
+
+def validate_transformer_tap_regulator(data: SingleDataset) -> List[ValidationError]:
```

Comment:
This function can be split into `validate_regulator` and `validate_transformer_tap_regulator` just to match the existing way.